alacritty_terminal/src/term/mod.rs

```diff
@@ -607,37 +612,76 @@ impl From<&BellConfig> for VisualBell {
 #[derive(Serialize, Deserialize, Debug, Copy, Clone, PartialEq)]
 pub struct SizeInfo {
     /// Terminal window width.
-    pub width: f32,
+    width: f32,
 
     /// Terminal window height.
-    pub height: f32,
+    height: f32,
 
     /// Width of individual cell.
-    pub cell_width: f32,
+    cell_width: f32,
 
     /// Height of individual cell.
-    pub cell_height: f32,
+    cell_height: f32,
 
     /// Horizontal window padding.
-    pub padding_x: f32,
+    padding_x: f32,
 
     /// Horizontal window padding.
-    pub padding_y: f32,
+    padding_y: f32,
+
+    /// Number of lines in the viewport.
+    screen_lines: Line,
+
+    /// Number of columns in the viewport.
+    cols: Column,
 
     /// DPR of the current window.
     #[serde(default)]
     pub dpr: f64,
```

Comment:
Seems strange that it was the only thing left as `pub`. Also, what do you think about changing dpr to f32, there's really not point in f64.

Reply:
It comes in as f64 and there's not a lot of places where we convert it to f32 either. So I don't see a reason to reduce the precision here?

This is pub because it must be set and read from the outside.